config_templates/gretel/synthetics/low-record-count.yml

```diff
@@ -1,20 +1,17 @@
-# For datasets that have a low number of records, in the hundreds or so.
-
-# Customize privacy filters https://docs.gretel.ai/gretel.ai/synthetics/synthetics-faqs/privacy-protection
+# deprecated: This configuration will be deprecated soon. Please use the "synthetics/tabular-actgan" configuration.
 
 schema_version: "1.0"
 name: "low-record-count"
 models:
   - actgan:
         data_source: __tmp__
         params:
-            epochs: 600
+            epochs: auto
             generator_dim: [1024, 1024]
             discriminator_dim: [1024, 1024]
             generator_lr: 0.0001
             discriminator_lr: .00033
-            batch_size: 600
-        # Gretel privacy filtering
+            batch_size: auto
         privacy_filters:
             outliers: auto
-            similarity: auto
+            similarity: auto
```

Comment:
github is complaining about not seeing a newline at the end of the file. Not sure if it's necessary, but I've been adding one just in case.